src/ActionBar/ActionBar.Component.js

```diff
@@ -94,11 +95,11 @@ const clickBackBtn = () => {
                 The **Action Bar** is located at the top of the page and is used for page title and main actions for the
                 page.
             </Description>
-            <Import sourceModulePath={require.resolve('./ActionBar')} />
+            <Import sourceModulePath={path.join(__dirname, './ActionBar')} />
```

Comment:
I am getting the following error when I attempt to run it in dev mode:
![screen shot 2019-02-14 at 8 42 17 am](https://user-images.githubusercontent.com/11407353/52793999-9eabc280-3034-11e9-9b01-c394cea31ac5.png)

I'm not sure if this fixes it in the webpack mode, but it definitely doesn't work in local dev mode.  Is it possible that we need two different solutions?

Reply:
@greg-a-smith  I got it working in dev mode by running `npm run build` then `npm start`, I think that's all it needs.

Reply:
We could add a node environment check before this and switch out `lib` for `src` and avoid having to run build first: https://github.com/SAP/fundamental-react/blob/master/src/_playground/documentation/Import/Import.js#L17

Reply:
I don't think a node environment check will work because we are running the docs in development mode to make sure `propTypes` don't get removed.

Reply:
But if doing a `npm run build` solves it, then maybe we just wire that into the `npm start` script.

Reply:
I was thinking along the same line as @greg-a-smith I'll add build to the start script.